simvue/run.py

```diff
@@ -716,8 +716,8 @@ def callback_function(status_code: int, std_out: str, std_err: str) -> None:
             identifier,
             *_pos_args,
             executable=executable,
-            script=script,
-            input_file=input_file,
+            script=f"{script}",
```

Comment:
These should not be converted to strings, and instead the function should expect a file path as an input